resources/strings/src/commonMain/kotlin/dev/sasikanth/rss/reader/resources/strings/DeTwineStrings.kt

```diff
@@ -0,0 +1,149 @@
+        buttonAll = "Alle",
+        buttonAddFeed = "Feed hinzufügen",
+        buttonGoBack = "Zurückgehen",
+        buttonCancel = "Abbrechen",
+        buttonAdd = "Hinzufügen",
+        buttonChange = "Erledigt",
+        feedEntryHint = "Feed-Link eingeben",
+        share = "Aktie",
+        scrollToTop = "Nach oben scrollen",
+        noFeeds = "Keine Feeds vorhanden!",
+        swipeUpGetStarted = "Wischen Sie nach oben, um zu beginnen",
+        feedNameHint = "Feedname",
+        editFeedName = "Bearbeiten",
+        errorUnsupportedFeed = "Der Link enthält keinen RSS/Atom-Feed.",
+        errorMalformedXml = "Der bereitgestellte Link enthält keinen gültigen RSS/Atom-Feed",
+        errorRequestTimeout =
+        "Zeitüberschreitung, überprüfen Sie Ihre Netzwerkverbindung und versuchen Sie es später erneut",
+        errorFeedNotFound = { "($it): Unter dem angegebenen Link wurde kein Inhalt gefunden." },
+        errorServer = {
+            "($it): Serverfehler. Bitte versuchen Sie es später noch einmal oder wenden Sie sich an den Website-Administrator."
+        },
+        errorTooManyRedirects =
+        "Die angegebene URL enthält zu viele Weiterleitungen. Bitte verwenden Sie eine andere URL.",
+        errorUnAuthorized = {
+            "($it): Sie sind nicht berechtigt, auf Inhalte unter diesem Link zuzugreifen."
+        },
+        errorUnknownHttpStatus = { "Inhalt konnte nicht mit HTTP-Code geladen werden: ($it)" },
+        postsSearchHint = "Beiträge durchsuchen",
+        searchSortNewest = "Neueste",
+        searchSortNewestFirst = "Das neuste zuerst",
+        searchSortOldest = "Älteste",
+        searchSortOldestFirst = "Die ältesten zu erst",
+        searchResultsCount = { count ->
+            when (count) {
+                1 -> "$count Ergebnis"
+                else -> "$count Ergebnisse"
+            }
+        },
+        bookmark = "Lesezeichen",
+        bookmarks = "Lesezeichen",
+        bookmarksPlaceholder = "Mit Lesezeichen versehene Beiträge werden hier angezeigt 🔖",
+        settings = "Einstellungen",
+        moreMenuOptions = "Weitere Menüoptionen",
+        settingsHeaderBehaviour = "Verhalten",
+        settingsHeaderFeedback = "Feedback und Fehlerberichte",
+        settingsHeaderOpml = "OPML",
+        settingsBrowserTypeTitle = "Verwenden Sie den In-App-Browser",
+        settingsBrowserTypeSubtitle =
+        "Wenn diese Option deaktiviert ist, werden Links in Ihrem Standardbrowser geöffnet.",
+        settingsEnableBlurTitle = "Aktivieren Sie die Unschärfe auf der Startseite",
+        settingsEnableBlurSubtitle =
+        "Wenn deaktiviert, werden Farbverläufe statt unscharfer Bilder angezeigt. Kann die Leistung verbessern.",
+        settingsShowUnreadCountTitle = "Anzahl der ungelesenen Beiträge anzeigen",
+        settingsShowUnreadCountSubtitle =
+        "Wenn diese Option deaktiviert ist, wird die Anzahl der ungelesenen Beiträge nicht im Feed-Blatt angezeigt",
```

Comment:
"Feed-Blatt" is not really clear to me. You likely mean the bottom bar with the feeds, but I would use "Feed-Übersicht" / "Feed overview".

Reply:
'feed selection' is the word would this surface "Futterauswahl"